packages/ra-core/src/sideEffect/accumulate.spec.js

```diff
@@ -0,0 +1,142 @@
+import { delay } from 'redux-saga';
+import { call, cancel, fork, put } from 'redux-saga/effects';
+import { createMockTask } from 'redux-saga/utils';
+
+import { accumulateFactory, finalizeFactory } from './accumulate';
+import { crudGetMany } from '../actions';
+
+describe('accumulate saga', () => {
+    describe('backward compatibility', () => {
+        it('proceeds with the accumulated action when no more actions are dispatched', () => {
+            const finalize = finalizeFactory({}, {});
```

Comment:
it feels weird not to use the same `tasks` and `accumulations` objects as for the other factory. I know you don't need it in this test, but that gives a wrong usage example.